samples/nodejs-typescript-repairshub/README.md

```diff
@@ -0,0 +1,81 @@
+# RepairsHub Copilot connector sample
```

Comment:
Let's add the `assets/sample.json` file so that the sample shows up in the Gallery at adoption.microsoft.com.